### DISCUSSION TITLE
razing the samplerate of the synthesizer

### DISCUSSION THREAD

**Discussion by king-dahmanus** [dectalk/dectalk#23] [General]
Hey fokes. Is there any interest in making the synth higher quality by razing the sample rate to at least 16khz?

**Comment by jaybird110127** [comment]
There has been some work on giving DECtalk a 22K option, however, last I heard, for various reasons it's not very feasible. The results, at least as of late last week, in my opinion are less than overwhelming, in that what mostly happens is that there are more highs on certain consonant sounds such as S's and T's, but not much else.

**Comment by king-dahmanus** [comment]
ohh I see, guess it needs major remodeling to make it work better.
